packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -273,9 +273,13 @@ export default {
     capitalizedTimestamp() {
       let displayDate = ''
       if (this.file.mdate) {
-        displayDate = this.formDateFromNow(this.file.mdate, 'Http')
+        displayDate = DateTime.fromRFC2822(this.file.mdate)
```

Comment:
Looks like it would be worth adding a `formDate` helper for absolute dates, right? You could accept the format as a param with `DateTime.DATETIME_FULL` as a default value.

Reply:
You mean just like `formDateFromNow()`, right? This method has a comment `// TODO: Refactor those into their own functions a.k.a relDateFromJSDate() etc`. What do think, let's do it that way and refactor `formDateFromNow()` in the course of this as well? I counted 5 usages, looks doable without too much effort.